src/common/Formatter.h

```diff
@@ -232,7 +232,7 @@ namespace ceph {
     void print_quoted_string(std::string_view s);
     void print_name(std::string_view name);
     void print_comma(json_formatter_stack_entry_d& entry);
-
+    void add_value(std::string_view name, double val);
```

Comment:
```suggestion
template <typename T, typename = std::enable_if_t<std::is_floating_point_v<T>>>
    void add_value(std::string_view name, T val);
```

Reply:
Shouldn't be it ambiguous if keeping both functions?
```
    template <typename T, typename = std::enable_if_t<std::is_floating_point_v<T>>>
    void add_value(std::string_view name, T val);

    template <typename T>
    void add_value(std::string_view name, T val);
```

Reply:
Ya, nevermind.